lib/features/login/domain/repository/account_repository.dart

```diff
@@ -6,5 +6,5 @@ abstract class AccountRepository {
 
   Future<void> setCurrentAccount(PersonalAccount newCurrentAccount);
 
-  Future<void> deleteCurrentAccount(String accountId);
+  Future<void> deleteCurrentAccount(String hashId);
```

Comment:
IMO, no problem with `accountId`. `hashId` make more confused

Reply:
accountId easily confused with AccountId in Session. hashId is just an identifier for a cached account. It can be tokenHashId or UserName